src/routes/NewUser/components/NewUserPage/NewUserPage.js

```diff
@@ -145,7 +145,11 @@ function NewUser() {
       return;
     }
 
-    const userUpdates = { ...values, ...userLocationInfo };
+    const userUpdates = {
+      ...values,
+      ...userLocationInfo,
+      browserNotifications: true,
```

Comment:
Added this so that new user page will enable the browser notifications of new users by default - they can then turn them off from the accounts page. @thebitguru is this what you were thinking or should this be removed?